src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerCFTTest.java

```diff
@@ -633,14 +638,15 @@ public void should_return_a_200_with_search_results_based_on_jurisdiction_locati
         common.insertTaskInCftTaskDb(taskVariables, "followUpOverdueReasonsForAppeal", headers);
 
         Response result = restApiActions.post(
-            ENDPOINT_BEING_TESTED + "?first_result=0&max_results=10",
+            ENDPOINT_BEING_TESTED,
             searchTaskRequest,
             headers
         );
 
         result.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("tasks.size()", lessThanOrEqualTo(10))
+            .body("tasks.size()", lessThanOrEqualTo(50)) //Default max results
+            .body("tasks.id", hasItem(taskId))
```

Comment:
With regards to asserting tasks.id, there was an issue in AAT a while back where the task was not returned within the limit required. Any chance this task could be 50 + n and therefore not returned?

Reply:
same as above.